### ISSUE TITLE
remove autocomplete in chrome browser

### ISSUE THREAD

**Issue by ggirodda** [shentao/vue-multiselect#728]
I want to thank you for your work, vue-multiselect works really great!
Can you please replace the autocomplete="off" with autocomplete="nope" as a bug fix for chrome wich ignores the first one ?
the bug
https://bugs.chromium.org/p/chromium/issues/detail?id=370363
the solution (on the bottom of the section "Disabling autocompletion")
https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion#Disabling_autocompletion
I tested it and it works well (for now)

**Comment by shentao** [comment]
Thanks! Sounds good :)

**Comment by vkruoso** [comment]
@shentao can you reopen this? It looks like it a PR was not opened. Also, it seems that the "nope" is ignored by Firefox and it reverts to default autocompletion behavior.

**Comment by Landen13** [comment]
@shentao 
When to expect release with this fix?

**Comment by laurens94** [comment]
It seems `autocomplete="nope"` doesn't work anymore in Chrome, it should be `autocomplete="off"` again..

**Comment by laurens94** [comment]
@shentao could you confirm this / reopen the issue?

**Comment by shentao** [comment]
I sooo hate that happening...

**Comment by michaelwills** [comment]
tl;dr I doubt there can be "one right way". How about a string option which defaults to `"off"`? My current ~hack~ solution right now, primarily because of Chrome, is to use the `@open` event to update the `autocomplete` attribute to `new-${id}-field` which has worked so far.

The long version:

It appears there _is_ an official answer. This explains the reasoning behind it ("think of the ~children~ users!")

https://bugs.chromium.org/p/chromium/issues/detail?id=468153#c164

so... `autocomplete="off"` no longer works as an option.

This says what to use:

https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofill

So there it says it's still to supposed to work but... no.

And the battle rages here:

https://bugs.chromium.org/p/chromium/issues/detail?id=587466

**Comment by vcarbo** [comment]
Any news about this fix ? I still have autocomplete="nope"

**Comment by chrislfield** [comment]
Can this be re-opened? Is a minor yet annoying usability bug.

**Comment by sontd-0882** [comment]
This is really annoy. It's still `nope` instead of `off`. I'm using `v2.1.6`

![Screen Shot 2020-01-31 at 2 41 40 PM](https://user-images.githubusercontent.com/49388081/73521284-3407b880-4438-11ea-84df-d5dd2a6645df.png)

**Comment by ravipatel2293** [comment]
I have resolved this issue by using refs.
Below is the code which i have written in mounted life hook.
```
// **multiselectRef** is ref of the multiselect componenet
mounted(){
       this.$refs.multiselectRef.$refs.search.setAttribute("autocomplete", "off")
}
```

**Comment by berk9595** [comment]
```
<input type="text" id="myid">

$( document ).ready(function() {
         $('#myid).attr("autocomplete","false");
         $('#myid').attr('readonly', 'true');
         $( "#myid" ).click(function() {
         $('#myid').attr('readonly', null);
  });
```

**Comment by ian-leggett** [comment]
`autocomplete="nope"` isn't even a thing, this is inaccessible, nope has no meaning and is not valid.

**Comment by juliennesme** [comment]
It seems autocomplete="off" won the battle...

This is still anoying. It should be reopened.

https://www.w3schools.com/howto/howto_html_autocomplete_off.asp
https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion
Just did a test on chrome it works with "off"

I'm implementing the hack from @ravipatel2293 but hope for a better long term solution
